nix/marlowe-cardano/deploy/README.md

```diff
@@ -25,7 +25,7 @@ To push an oci-image to the registry you will need to get access credentials, by
 Then to push, for example, marlowe-chain-sync run:
 
 ``` sh
-nix run .\#oci-images.x86_64-linux.marlowe-chain-sync.copyToRegistry
```

Comment:
I n Recently we upgraded `std` and so the `copyToRegistry` attribute is no longer present. I'm not familiar with oci-images so I need a little help here

Reply:
Maybe https://nixos.wiki/wiki/Docker?